lib/llm/cohere.rb

```diff
@@ -13,6 +10,8 @@ class Cohere < Base
     }.freeze
 
     def initialize(api_key:)
+      require_optional_dependency "cohere-ruby"
```

Comment:
Do you think `require_optional_dependency()` is a better method name than `require_dependency()`? Just based on the fact that when the `initialize()` method is called, it's technically a "required dependency"?